styleguide/source/assets/scss/01-atoms/_table.scss

```diff
@@ -1,116 +1,92 @@
-/**
- * @file
- * Define responsive table.
- *
- *
+/*
+ * jQuery Basic Table
+ * Author: Jerry Low
  */
 
 table {
-  display: block;
-  width: 100%;
+  background: $white;
   border-collapse: collapse;
-  border-spacing: 0;
-  border: 1px solid $gray-64;
-  table-layout: fixed;
+  width: 100%;
+}
 
-  @include breakpoint($bp-small) {
-    display: table;
-  }
+table tr,
+table th,
+table td {
+  border: none;
 }
 
-caption {
-  display: block;
+table th,
+table td {
+  text-align: left;
+  padding: $gutter/4;
+
+}
+
+table th {
+  text-transform: uppercase;
+}
+
+table tr td {
+  background: $gray-20;
+}
+
+table tr:nth-of-type(2n+2) td {
```

Comment:
According to Zeplin Tables: https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5ac65d98850afa36779ea342 

- The gray background table cell starts, followed by the white background table cells
- The tables and their cells have a slight gray border 
- The header text for each column needs to be bold

Do these need to match up?

Reply:
@russom-woldezghi the zeplin file you're referring to is out of date
Here is the updated version: https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/5ac3997560cdf96b3b16ea03